### DISCUSSION TITLE
Brand Name alongside logo

### DISCUSSION THREAD

**Discussion by TheFrankman** [filamentphp/filament#9717] [General]
Hello There,

Pretty basic question. I'd like to display the brand name along side the logo in the top navigation.

The easy way to do this is to create a `resources/views/vendor/filament-panels/components/sidebar/index.blade.php` and add it but I feel like there is probably a more filament approach.

I have checked that available render hooks and can't find anything appropriate.

This is my first filament site so forgive me if I am missing something obvious here.

**Comment by leandrocfe** [comment]
You can use a view
```php
use Filament\Panel;
 
public function panel(Panel $panel): Panel
{
    return $panel
        // ...
        ->brandLogo(fn () => view('filament.admin.logo'));
}
```

`filament()->getBrandName()` inside the view

**Reply by TheFrankman** [reply]
Perfect, thank you!
